com.unity.netcode.gameobjects/Tests/Editor/Transports/UnityTransportTests.cs

```diff
@@ -126,5 +126,40 @@ public void UnityTransport_RestartSucceedsAfterFailure()
 
             transport.Shutdown();
         }
+
+#if UTP_TRANSPORT_2_0_ABOVE
```

Comment:
Because all encryption support is behind the `UTP_TRANSPORT_2_0_ABOVE` define. (Technically that's not necessary since DTLS is supported in 1.3 with the same APIs used for 2.0, but if we are to support encryption no matter the version of UTP, I'd say that's a larger change that would warrant its own PR.)